templates/emails/invitation-for-cfp-template.md

```diff
@@ -0,0 +1,16 @@
+## Template for the invitation for CFP
+
+Hello {{POTENIAL_SPEAKER_FIRST_NAME}},
+
+We hope you are in good health and doing well. This email is regarding PyCon India 2020 inviting you to submit a proposal for our open call for proposals. PyCon India is the premier conference for the Python programing language in India and has been running for well over 12 years, scaling in both size and folks attending. Last year, we had over 300+ proposals! What's better, we are completely non-profit and volunteer-run through and through. Helping promote Python through many communities around India. 
+
+This year with the current situation [1] We, the conference organizers, are taking this as a unique opportunity to be able to put more efforts into making this year's conference bigger, more inclusive, and much more accessible to our lovely attendees from all over the world in ways more than one. And, we need your expertise in that. We would be glad to see your proposal at our conference. We believe your experience and insights with Python and/or with Python community will be a critical part of making this conference a huge success.
```

Comment:
```suggestion
This year, with the current situation [1], we, the conference organizers, are taking this as a unique opportunity to be able to put more efforts into making this year's conference bigger, more inclusive, and much more accessible to our lovely attendees from all over the world in ways more than one. And, we need your expertise in that. We would be glad to see your proposal at our conference. We believe your experience and insights with Python and with Python community will be a critical part of making this conference a huge success.
```

Something doesn't sound right with the first sentence. It is too long and has too much content. Can we split it?